src/Header.js

```diff
@@ -1,9 +1,45 @@
 import React, { Component } from 'react';
+import DefaultService from './service/default.service.js';
+import './header.css';
 
 class Header extends Component {
+
+    constructor(props) {
+        super(props);   
+        this.service = props.service; 
+        this.state = {
+            login: '<?>'
```

Comment:
I would call this loggedIn or authenticated, rather than login which is a verb.  State should be adjectives and nouns.

Reply:
unless this is a function being passed in... sorry, just started the review so I wasn't sure.